snippets/Check_for_palindrome.md

```diff
@@ -0,0 +1,10 @@
+### Check For Palindrome
+
+```
+function palindrome(str) {
+  var rg =/[\W_]/g;
+  var low_rep=str.toLowerCase().replace(rg,'');
+  var final= low_rep.split('').reverse().join('');
+  return final == low_rep;
```

Comment:
All of this can be a one liner like `str.toLowerCase().replace(/[\W_]/g,'').split('').reverse().join('') === str` (double-check if this works).

Reply:
I double checked, so see the latest commit.